src/steamship/data/operations/generator.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+from typing import List, Optional
+
+from steamship.base.request import Request
+from steamship.base.response import Response
+from steamship.data.block import Block
+
+
+class GenerateRequest(Request):
```

Comment:
any reason not to include a `kwargs` style extra bit here?  I feel like we need a way to pass runtime args to calls to generate.

Reply:
the text example was "stop" keywords

Reply:
NOTE TO FUTURE SELF: we will address side-channel info in a separate PR.